assets/src/js/frontend/give-stripe-elements.js

```diff
@@ -308,15 +308,24 @@ class GiveStripeElements {
 	 * @param setupStripeElement
 	 * @param cardElements
 	 *
+     * @unreleased Scrolls Stripe checkout modal into view for all screen sizes.
+     *
 	 * @since 2.8.0
 	 */
 	triggerStripeModal( formElement, stripeElements, setupStripeElement, cardElements ) {
 		const idPrefixElement = formElement.querySelector( 'input[name="give-form-id-prefix"]' );
 		const stripeModalDonateBtn = formElement.querySelector( `#give-stripe-checkout-modal-donate-button-${ idPrefixElement.value }` );
 		const cardholderName = formElement.querySelector( 'input[name="card_name"]' );
+        const stripeModalContent = document.querySelector('.give-stripe-checkout-modal-container')
+        const purchaseButton = document.querySelector('#give-purchase-button')
 		const completeCardElements = {};
 		let completeCardStatus = false;
 
+        // Scroll checkout modal container into view.
+        purchaseButton.addEventListener('click', function(){
+            stripeModalContent.scrollIntoView({ behavior: 'smooth' });
+        })
```

Comment:
Formatting looks a touch off. Is prettier formatting doing its thing?

Reply:
Yes, this was formatted by prettier. 

https://user-images.githubusercontent.com/75056371/235512220-307d420c-f962-4c44-b4d6-5bf9fed6f720.mov